authors/apps/authentication/tests/test_verify_user.py

```diff
@@ -8,19 +8,37 @@ class TestUserVerification(BaseTest):
 
     def test_successfull_verification(self):
         token = self.generate_jwt_token('pablo@escobar.com', 'Escobar')
-        url = "/api/users/activate/" + token
+        url = self.verification_of_user + token
 
         response = self.client.get(url)
         self.assertEqual(response.data['message'], "You have been verified successfully")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         
-    def test_unsuccessfull_verification(self):
+    def test_invalid_verification_token(self):
         self.signup_user('Elchapo', 'el@chapo.com', 'Maina9176')
         self.verify_user('el@chapo.com')
         token = self.generate_jwt_token('el@chapo.com', 'Elchapo')
-        url = "/api/users/activate/" + token
+        url = self.verification_of_user + token
 
         response = self.client.get(url)
         
         self.assertEqual(response.data['message'], "Your email is already verified")
-        self.assertEqual(response.status_code, status.HTTP_406_NOT_ACCEPTABLE)
\ No newline at end of file
+        self.assertEqual(response.status_code, status.HTTP_406_NOT_ACCEPTABLE)
+
+    def test_invalid_verification_token(self):
+        token = "sidvfldifhsv"
+        url = self.verification_of_user + token
+
+        response = self.client.get(url)
+        self.assertEqual(response.data['detail'], "Invalid Token. The token provided cannot be decoded!")
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_expired_jwt_token(self):
+        token = self.generate_expired_token()
+        url = self.verification_of_user + token
+
+        response = self.client.get(url)
+        self.assertEqual(
+            response.data['detail'],
+            'The token used has expired. Please authenticate again!')
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
```

Comment:
Hello @Bakley ,
Can you add new line at the end of the file

Reply:
Added